client/wfnews-war/src/main/angular/src/app/components/common/circle-icon-button/circle-icon-button.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  // eslint-disable-next-line @angular-eslint/component-selector
```

Comment:
Seeing this in a few places, can you let me know the use for it? from a quick search it disables the next line, I'm just curious as to the use case here. Thanks